rgbd/test/test_client_templ.h

```diff
@@ -31,6 +41,28 @@ template<class T>
 int main_templ(int argc, char **argv)
 {
     ros::init(argc, argv, "rgbd_transport_test_client");
+
+    bool headless = false;
+    std::string arg;
+    for (uint i=1; i<argc; ++i)
+    {
+        arg = argv[i];
+        if (arg == "--headless")
+        {
+            headless = true;
+            ROS_INFO("Running in headless mode");
+        }
+        else if(arg == "--help")
+        {
+            tue::filesystem::Path path(argv[0]);
+            usage(path.filename());
```

Comment:
Isnt there a more clever way of doing this? Since now we introduce a dirty dependency on the tue filesystem.

Reply:
Not a more clever way. Only a more dirty way. Which is doing some string magic ourselves. But the tue_filesystem function works for windows and ubuntu. Will our string magic would only work on ubuntu.

Reply:
We could make usage something like "rgbd_test_client_[type] [--headless]" so that we do not need to specify the exact executable name.

Reply:
That would be the lazy version. I can work with that